string rosbag
---
bool success
---
TransferStatus status
